dbms/src/Flash/Coprocessor/DAGExpressionAnalyzer.cpp

```diff
@@ -105,6 +105,24 @@ DataTypePtr findDuplicateAggFunc(
     return nullptr;
 }
 
+/// return `duplicated window function`->getReturnType if duplicated.
+/// or not return nullptr.
+DataTypePtr findDuplicateWindowFunc(
```

Comment:
maybe wrap findDuplicateWindowFunc and findDuplicateAggFunc into one function since they have the same logic.

Reply:
It doesn't seem easy to merge into one function because of the difference between `window_functions_description.window_function` and `aggregated.function`

Reply:
merged to `findDuplicateAggWindowFunc`